app/src/main/java/ch/hslu/refashioned/ui/color/GradientProvider.java

```diff
@@ -7,14 +7,12 @@
 import java.util.stream.Stream;
 
 import ch.hslu.refashioned.model.history.Brand;
+import ch.hslu.refashioned.model.sustainability.Score;
 
 public final class GradientProvider {
     @SuppressWarnings("OptionalGetWithoutIsPresent")
     public static ColorFactory getScoreGradient() {
-        var min = Stream.of(Brand.values()).mapToInt(b -> b.getScore().getOverall()).min().getAsInt();
-        var max = Stream.of(Brand.values()).mapToInt(b -> b.getScore().getOverall()).max().getAsInt();
-
-        return getScoreGradient(min, max);
+        return getScoreGradient(Score.MIN_SCORE, Score.MAX_SCORE);
```

Comment:
Ich fänds persönlich schöner, wenn immer der beste Brand vorgibt welcher Wert der beste ist. Mit 49 als max. hätten wir nun immer nur "Gelb" und darunter...das war recht bewusst so gewählt hier

Reply:
Kann ich für den max-Wert verstehen. Jedoch kann ein Wert auch unter den min-Wert aller verfügbaren fallen (wie in deinem Mock) und dann hätte dieser mit dem aktuellen Fix die Farbe schwarz anstelle rot. Sprich das Minimum muss aus meiner Sicht aktuell konstant 0 sein.